anthos-bm-gcp-bash/README.md

```diff
@@ -0,0 +1,123 @@
+## Anthos Baremetal on Google Compute Engine VMs
+
+This sample shows you how to setup an Anthos clusters on bare metal in High
+Availability (HA) mode using Compute Engine Virtual Machines (VMs). The
+[setup_and_install_abm](./setup_and_install_abm.sh) script encapsulates all
+the steps required to setup the Compute Engine VMs and to trigger the
+installation of Anthos on bare metal. Follow the
+[Try Anthos clusters on bare metal on Compute Engine VMs](https://cloud.google.com/anthos/clusters/docs/bare-metal/latest/try/gce-vms) guide for a step by step
+explanation of all the steps included in this script.
+
+
+### Pre-requisites
+
+- A workstation with access to internet _(i.e. Google Cloud APIs)_ with the following installed
+  - [Git](https://www.atlassian.com/git/tutorials/install-git)
```

Comment:
Why is this pointing to Atlassian instead of the official git site? (https://git-scm.com/)

Reply:
Fixed in f0b579990a1f1a2db9f1e1ef51c476e1ffe22d56